internal/controllers/device_config_controller.go

```diff
@@ -167,9 +174,9 @@ func (d *DeviceConfigController) GetDeviceSettingsByTemplate(c *fiber.Ctx) error
 	templateName := c.Params("templateName")
 
 	// Query the database to get the Device Settings based on the template name using SQLBoiler
+
```

Comment:
nit: remove empty line btw comment and code here

Reply:
done